docs/user-guide/networkpolicies.md

```diff
@@ -0,0 +1,82 @@
+---
+---
+
+* TOC
+{:toc}
+
+## What is a _Network Policy_?
+
+A Network Policy is a specification of how groupings of pods are allowed to communicate with each other and other network endpoints.
+
+NetworkPolicy resources use labels to select pods and define whitelist rules which allow traffic to the selected pods in addition to what is allowed by the ingress isolation policy for a given namespace.
+
+## Prerequisites
+Before you start using the NetworkPolicy resource, there are a few things to understand.  The NetworkPolicy resource is a beta resource and is 
```

Comment:
Remove this paragraph; we always doc "the state of the art" (forget the past), and this just adds another thing we'll have to update later.